pkg/manager/manager.go

```diff
@@ -112,18 +111,18 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 		log.Info("Setting up controllers")
 		err = controller.AddToManager(k.runtimeManager, poolManager)
 		if err != nil {
-			errors.Wrap(err, "unable to register controllers to the manager")
+			return errors.Wrap(err, "unable to register controllers to the manager")
 		}
 
 		log.Info("Setting up webhooks")
 		err = webhook.AddToManager(k.runtimeManager, poolManager)
 		if err != nil {
-			errors.Wrap(err, "unable to register webhooks to the manager")
+			return errors.Wrap(err, "unable to register webhooks to the manager")
 		}
 
 		err = k.runtimeManager.Start(k.restartChannel)
 		if err != nil {
-			errors.Wrap(err, "unable to run the manager")
+			log.Error(err, "unable to run the manager")
```

Comment:
Why don't we want to return here ?

Reply:
cases such as kubevirt was just applied and we want to restart the manager without restarting the pod